src/data/content/design-system/come-iniziare/per-sviluppatori.yaml

```diff
@@ -262,4 +235,34 @@ components:
               align: middle
               hidden: true
               size: sm
+              addonClasses: ms-2
+
+        - name: TextImageCta
+          # title: Design tokens
+          headingLevel: 4
+          noSpace: true
+          text: |
+            Nei prossimi mesi inizieranno i lavori per integrare i design token di UI Kit Italia in Bootstrap Italia e nelle altre risorse di sviluppo. Puoi già usarli nei tuoi progetti o seguire i lavori nel repository dedicato al progetto Design Tokens Italia.
+          ctas:
+          - label: Vai al repository dedicato #C
+            screenReaderText: " Design Tokens Italia (si apre in una nuova finestra)"
+            url: https://github.com/italia/design-tokens-italia #M
+            blank: true #M #C true if new tab for external links
+            icon:
+              icon: sprites.svg#it-github #C #I #it-arrow-right | it-external-link for external links
+              color: primary
+              align: middle
+              hidden: true
+              size: sm
+              addonClasses: ms-2
+          - label: Changelog #C
+            screenReaderText: " Design Tokens Italia (si apre in una nuova finestra)"
```

Comment:
```suggestion
            screenReaderText: " Design Token Italia (si apre in una nuova finestra)"
```

Reply:
@zetareticoli ha la `s` nella repository, sia lo slug che il titolo nel README...

Reply:
Fatto salvo che in italiano non dovrebbe averlo il plurale, come sappiamo, è vero che i nomi repository seguono una logica "più inglese". Lascio a te e @danielaiozzo decidere, ma in caso va corretto dovunque, anche nel fondamento Design Tokens. Forse fatto salvo gli slug che non cambierei ne nelle repo ne qui nella documentazione in caso.

Reply:
anch'io modificherei tutto al singolare, anche perché anche nel fondamento (a eccezione che nel titolo) viene usato "design token" e non "design tokens"

Reply:
fate voi?

Reply:
Nel testo concordo, sul nome però del progetto GitHub "Design Tokens Italia" secondo me no, rimane inglese. Anche perché è l'url della repo.